recipes/tests.rb

```diff
@@ -136,11 +136,27 @@
       command "rpm -qa | grep libfabric && rpm -qa | grep efa-"
       user node['cfncluster']['cfn_cluster_user']
     end
+    execute 'check intel mpi installed' do
+      command "rpm -qa | grep intel-mpi"
+      user node['cfncluster']['cfn_cluster_user']
+    end
+    execute 'check intel mpi version' do
+      command "module load intelmpi && mpirun --help | grep 'Version 2019 Update 6'"
+      user node['cfncluster']['cfn_cluster_user']
+    end
   when 'ubuntu1604', 'ubuntu1804'
     execute 'check efa rpm installed' do
       command "dpkg -l | grep libfabric && dpkg -l | grep 'efa '"
       user node['cfncluster']['cfn_cluster_user']
     end
+    execute 'check intel mpi installed' do
+      command "dpkg -l | grep intel-mpi"
+      user node['cfncluster']['cfn_cluster_user']
+    end
+    execute 'check intel mpi version' do
```

Comment:
minor: you could move this outside of when and avoid the duplication

Reply:
done